business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/ProjectIntegrationTest.java

```diff
@@ -60,6 +61,7 @@ public static void createRepository() {
         createSpace(SPACE);
     }
 
+    @Ignore
```

Comment:
please add here a reason and jira URL you created for fixing the tests in future